glfw31-gl41core-triangle/triangle.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"fmt"
+	"runtime"
+
+	gl "github.com/go-gl/gl/v4.1-core/gl"
+	glfw "github.com/go-gl/glfw/v3.1/glfw"
+)
+
+const windowWidth = 800
+const windowHeight = 640
+
+var vertexShader = `
+#version 410
+
+in vec3 pos;
+out vec3 position; // used in fragment shader
+
+void main() {
+  position = pos;
+  gl_Position = vec4(pos, 1.0);
+}
+` + "\x00"
+
+var fragmentShader = `
+#version 410
+
+in vec3 position; // comes from vertex shader
+out vec4 outColor;
+
+void main() {
+  outColor = vec4(position+0.5, 1.0); // adds 0.5 to make it pretty (shifts the colors)
+}
+` + "\x00"
+
+func main() {
+	runtime.LockOSThread()
+
+	initGlfw()
+	setOpenGlVersion()
+	var win = makeWindow()
+	printOpenGlVersionInfo()
+
+	var vao = createVao()
+
+	gl.Viewport(0, 0, windowWidth, windowHeight)
+
+	shader := createProgram()
```

Comment:
tiny nit: Use of `:=` is inconsistent with use of `var` above, consider using the same format